helm/odigos/values.yaml

```diff
@@ -328,6 +328,11 @@ odiglet:
   # into the Odiglet to ensure it can access the container runtime unix socket.
   customContainerRuntimeSocketPath: ''
 
+  # run the odiglet container as privileged.
+  privileged: true
+  allowHostPathMounts: true
+  hostPid: true
```

Comment:
consider making these so the default "false".

e.g. when they are missing (default), it's like they are false